Ansible/collection/jfrog/ansible/galaxy.yml

```diff
@@ -3,40 +3,43 @@
 # The namespace of the collection. This can be a company/brand/organization or product namespace under which all
 # content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
 # underscores or numbers and cannot contain consecutive underscores
-namespace: jfrog
+namespace: "jfrog"
 
 # The name of the collection. Has the same character restrictions as 'namespace'
-name: ansible
+name: "ansible"
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.0.0
+version: "1.0.4"
 
 # The path to the Markdown (.md) readme file. This path is relative to the root of the collection
-readme: README.md
+readme: "README.md"
 
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
 # @nicks:irc/im.site#channel'
 authors:
-- your name <example@domain.com>
+  - "Jeff Fry <jefff@jfrog.com>"
 
 
 ### OPTIONAL but strongly recommended
 
 # A short summary description of the collection
-description: your collection description
+description: "This collection provides roles for installing Artifactory and Xray. Additionally, it provides optional SSL and Postgresql roles if these are needed for your deployment."
 
 # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
 # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
 license:
-- GPL-2.0-or-later
+  - "GPL-2.0-or-later"
```

Comment:
Have we checked with Mark / Legal about which license to use?

Reply:
Our other public repos (like charts, cli)  are using Apache-2.0 so I changed to that.

https://github.com/jfrog